swarm/network/stream/syncer.go

```diff
@@ -130,12 +130,12 @@ func (s *SwarmSyncerServer) SetNextBatch(from, to uint64) ([]byte, uint64, uint6
 			// This is the most naive approach to label the chunk as synced
 			// allowing it to be garbage collected. A proper way requires
 			// validating that the chunk is successfully stored by the peer.
-			err := s.netStore.Set(context.Background(), chunk.ModeSetSync, d.Address)
-			if err != nil {
-				metrics.GetOrRegisterCounter("syncer.set-next-batch.set-sync-err", nil).Inc(1)
-				log.Debug("syncer pull subscription - err setting chunk as synced", "correlateId", s.correlateId, "err", err)
-				return nil, 0, 0, nil, err
-			}
+			//err := s.netStore.Set(context.Background(), chunk.ModeSetSync, d.Address)
```

Comment:
@zelig we should discuss why we do this.

Reply:
@acud we discussed this on the call. Previously as the comment states above, this was a hack to mark that chunks have been synced (even though we have no feedback for that in pull syncing).

Now that we are adding `push syncing`, we get an explicit `receipt` that a chunk has been received at the nearest node, so we know it has been synced. So marking that a chunk is synced is no longer needed here.

I will remove this code, it doesn't have to be commented out.